app/views/course/admin/sidebar_settings/edit.html.slim

```diff
@@ -0,0 +1,15 @@
+= page_header
+
+= simple_form_for @settings, url: course_admin_sidebar_path(current_course) do |f|
+  table.table.table-hover
+    thead
+      th = t('.name')
+      th = t('.weight')
+    tbody
+      = f.simple_fields_for :sidebar_items do |item_form|
+        tr
+          = item_form.hidden_field :id
```

Comment:
In the final output html it appears under tr, think it's correct ? 

And if I put this in a th/td, there will be a extra empty column in table.